internal/tool/generate/generator.go

```diff
@@ -86,13 +89,21 @@ Examples:
   weaver generate ./foo
 
   # Generate code for all packages in all subdirectories of current directory.
-  weaver generate ./...`
+  weaver generate ./...
+
+  # Generate code for all files that have a "//go:build good" line at the top of
+  the file.
+  weaver generate -tags good
+
+  # Generate code for all files that have a "//go:build good,prod" line at the
+  top of the file.
+  weaver generate -tags good,prod`
 )
 
 // Options controls the operation of Generate.
 type Options struct {
-	// If non-nil, use the specified function to report warnings.
-	Warn func(error)
+	Warn      func(error) // If non-nil, use the specified function to report warnings
+	BuildTags string
```

Comment:
It seems like the BuildTags fields includes the "-tags=" prefix. That seems a bit wrong to me. Could just contain the tags, and we would add the -tags= prefix below when setting BuildFlags:

```
cf.BuildFlags = []string{"-tags", opt.BuildTags}
```

Reply:
Good point. Fixed